commit 46a89fb0319d673b3139a068e3d89aed9f44fc16
Author: Christoph Moench-Tegeder <cmt@burggraben.net>

    use gdk legacy cursor interface by default
    
    Upstream https://bugzilla.mozilla.org/show_bug.cgi?id=1876366#c16
    hints that we could re-test this once we have GTK 3.24.42 (with the
    now-current gtk3-3.24.41 the original problem is still reproducable,
    but toggling this flag does fix it for me)

diff --git modules/libpref/init/StaticPrefList.yaml modules/libpref/init/StaticPrefList.yaml
index 835450712a12..f2249006c36d 100644
--- modules/libpref/init/StaticPrefList.yaml
+++ modules/libpref/init/StaticPrefList.yaml
@@ -16070,7 +16070,7 @@
 # Whether to use gtk legacy cursor API.
 - name: widget.gtk.legacy-cursors.enabled
   type: bool
-  value: false
+  value: true
   mirror: always
 
 # Whether to use gtk high contrast themes to disable content styling like on
